pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -474,30 +462,24 @@ func (ctrl *PersistentVolumeController) syncBoundClaim(claim *v1.PersistentVolum
 			// volume yet. We can't distinguish these cases.
 			// Bind the volume again and set all states to Bound.
 			klog.V(4).Infof("synchronizing bound PersistentVolumeClaim[%s]: volume is unbound, fixing", claimToClaimKey(claim))
-			if err = ctrl.bind(volume, claim); err != nil {
-				// Objects not saved, next syncPV or syncClaim will try again
-				return err
-			}
-			return nil
+			// Objects not saved, next syncPV or syncClaim will try again
```

Comment:
My concern is that the lack of if statement now makes it look as if this `// Objects not saved...` comment is always true. It is only the case if the `ctrl.bind` command fails (i.e. returns an error).

Could you update the comment to indicate that?